gettsim/pensions.py

```diff
@@ -121,14 +123,11 @@ def _lohnkomponente(tb_pens, yr):
     the previous years. For details see
     https://de.wikipedia.org/wiki/Rentenanpassungsformel
     """
-    return tb_pens.loc["meanwages", yr - 1] / (
-        tb_pens.loc["meanwages", yr - 2]
+    return tb_pens["meanwages"][yr - 1] / (
```

Comment:
That is way to complicated for a return statement. Compute intermediate values with good variables names.